ballerina-test/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Shall we include License header?

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-distribution/pull/444/commits/6f025ca59e8db4293a7ca8655a149b66e3aef1d2